database/api/routes.md

```diff
@@ -1,123 +0,0 @@
-# api/v1/save/
```

Comment:
we should keep this file, usually api's have some sort of documentation

Reply:
I added this back